Purchases/Networking/Backend.swift

```diff
@@ -495,14 +495,17 @@ private extension Backend {
         }
 
         do {
-            let customerInfo = try CustomerInfo(data: response)
+            let customerInfo = try parseCustomerInfo(fromMaybeResponse: maybeResponse)
```

Comment:
This makes sense to me 👍 but I don't have all the context here, I'll leave others to take a look.